manage.py

```diff
@@ -1,9 +1,45 @@
-#!/usr/bin/env python
 """Django's command-line utility for administrative tasks."""
-import os
 import sys
+import os
+from contextlib import nullcontext, contextmanager
```

Comment:
An alternative solution to importing coverage is
```suggestion
from contextlib import contextmanager, nullcontext, suppress

with suppress(ImportError):
    from coverage import Coverage
```
This way, it wouldn't be imported in the function context, which may be perceived as a bit more elegant.
Though, the downside is that the import would be attempted even when unneeded (for non-test subcommands).